buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestTestPlugin.groovy

```diff
@@ -32,6 +32,7 @@ import org.gradle.api.Project
 public class RestTestPlugin implements Plugin<Project> {
     List REQUIRED_PLUGINS = [
         'elasticsearch.build',
+        'elasticsearch.rest-test-with-dependencies',
```

Comment:
Thanks for pointing this out. I actually think that "test-with-dependencies" should not be in that list anymore. As it has no relation to the RestTestPlugin.